### PR TITLE
Finished home page for youtube

### PR DESCRIPTION
**Pull request by DanCarl857**

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    margin-top: 5em;
+}
+
+main {
+    margin: 0 1.8em 0 1.8em;
+    display: grid;
+    grid-gap: 1.5em;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 0.2fr 1fr;
+   grid-template-areas: "video video aside"
```
Comment (efrapp) [inline styles/main.css]: Try always to get the lines aligned.

File: styles/main.css [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    padding: 0 10px 0 10px;
+    margin-bottom: 10px;
+    position: fixed;
+    top: -0.2em;
+    width: 100%;
+    z-index: 1;
+    background: white;
+}
+
+.header__logo {
```
Comment (efrapp) [inline styles/main.css:20]: Why do you use `__` syntax to name the classes? does it have an special meaning?

File: styles/main.css [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+  height: 0; 
+  overflow: hidden;
+}
+ 
+.main__video-container iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```
Comment (efrapp) [inline styles/main.css]: Be consistent with the indentation, some times you use 1 space and other you use more than one.

File: styles/main.css [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+}
+ 
+@media only screen and (max-width: 1002px)  {
+     main {
+        margin: 0 1.8em 0 1.8em;
+        display: grid;
+        grid-gap: 1.5em;
+         grid-template-columns: 1fr;
+        grid-template-rows: 0.2fr 1fr 0.55fr;
+       grid-template-areas:       "video"
```
Comment (efrapp) [inline styles/main.css]: Also, check the indentation here.

**Review by efrapp — CHANGES_REQUESTED** [review]
# The page looks nice!

The page is pretty close to the original but please address the followings suggestions:

1. Check this issue: #2 
2. Add a live version of the project, for that you can use [Github pages](https://pages.github.com/)
3. Add descriptive information to the project, what is the purpose? what concepts did you apply? and the author.
4. Check [HTML validator](https://validator.w3.org/) to the `index` page

You made a great job organizing the files in folders and the indentation is well used.

Please address the suggestions and send a code review again.

**Comment by DanCarl857** [comment]
thanks for the review @efrapp

**Review by efrapp — APPROVED** [review]
# You are done! 👍 

All the changes were made, so you can continue. I want to insist on the benefits of having a descriptive Readme. Bear in mind other people can see your work and good documentation is always needed.

No need to send a code review again.

I wish you the best in your path!

**Pull request merged**